WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -3564,27 +3566,25 @@ e_byzantium = {
 		
 		capital = 560 # Cherson
 		
-		hellenic_pagan = 600
-		bosporan_faith = 200
+		hellenic_pagan = 200
+		bosporan_faith = 600
```

Comment:
How was Tauricia, in any way, _religiously_ important to the Hellenic faith? I know why it's important to Rome in secular terms, obviously, but not to the Hellenic _religion_.